src/Sylius/Bundle/AttributeBundle/Form/Type/AttributeValueType.php

```diff
@@ -13,6 +13,7 @@
 
 namespace Sylius\Bundle\AttributeBundle\Form\Type;
 
+use Sylius\Bundle\CoreBundle\Form\DataTransformer\ResourceToIdentifierCacheableTransformer;
```

Comment:
AttributeBundle cannot depend on CoreBundle.

Please either open PR for ResourceBundle with `ResourceToIdentifierCacheableTransformer` or somehow extend this `AttributeValueType` at Core.

Reply:
I'm good with PR to ResourceBundle. It's here: https://github.com/Sylius/SyliusResourceBundle/pull/771/files